lib/cocoapods/installer/target_installer/aggregate_target_installer.rb

```diff
@@ -107,11 +107,14 @@ def resources_by_config
         end
         resources_by_config = {}
         target.user_build_configurations.keys.each do |config|
-          file_accessors = library_targets.select { |t| t.include_in_build_config?(target_definition, config) }.flat_map(&:file_accessors)
-          resource_paths = file_accessors.flat_map { |accessor| accessor.resources.flat_map { |res| res.relative_path_from(project.path.dirname) } }
-          resource_bundles = file_accessors.flat_map { |accessor| accessor.resource_bundles.keys.map { |name| "${BUILT_PRODUCTS_DIR}/#{name.shellescape}.bundle" } }
-          resources_by_config[config] = (resource_paths + resource_bundles).uniq
-          resources_by_config[config] << bridge_support_file if bridge_support_file
+          library_targets.select { |t| t.include_in_build_config?(target_definition, config) }.each do |pod_target|
+            file_accessors = library_targets.select { |t| t.include_in_build_config?(target_definition, config) }.flat_map(&:file_accessors)
```

Comment:
This must be now only `file_accessors = pod_target.file_accessors`. The rest happens already in the line before.